polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -225,7 +225,6 @@ $loading-panel-height: 53px;
 
 .TableHeading {
   background: var(--p-color-bg-subdued);
-  z-index: var(--p-z-index-1);
```

Comment:
Do we have an inclination of why this was here in the first place?

Reply:
Looks like it was added to support Tooltips on header content PR #8277 (this [line here](https://github.com/Shopify/polaris/pull/8277/files#diff-e9d0c2a006c030f315115b92e46c3b44667bef0b5e909e1b3af852e6eb2b605bR214)).

I verified that the Tooltips on heading content (sticky and non-sticky header) still work using the Storybook examples "With Sortable Custom Headings" and "With Custom Tooltips" 👍